src/pages/LandingPage/landingpage-style.js

```diff
@@ -1,22 +1,69 @@
 import styled from "styled-components";
+import Button from "../../components/Button";
 
 const nhfBlue = "#4a90e2";
 // const nhfYellow = "#e7ff00";
 
+export const LandingPageComponent = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const LandingPageContainer = styled.div`
+  width: 100%;
+  background-color: #4a90e2;
+  display: flex;
```

Comment:
why are you setting display: flex in all divs.

is this not needed on only the parent div.